ui/imports/shared/controls/WalletAccountListItem.qml

```diff
@@ -0,0 +1,98 @@
+import QtQuick 2.15
+
+import StatusQ 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+
+import AppLayouts.Wallet 1.0
+
+import utils 1.0
+
+StatusListItem {
+    id: root
+
+    property bool clearVisible: false
+
+    required property string name
+    required property string address
+    required property string chainShortNames
+    required property string emoji
+    required property string walletColor
+    required property var currencyBalance
+    required property string walletType
+    required property bool migratedToKeycard
+    property var accountBalance: null
```

Comment:
It would be very helpful for other developers to have description what's the expected structure of that property (and one above). Otherwise reading api is not sufficient, people need to search in the impl.